testData/solutions/linux.json

```diff
@@ -1430,15 +1430,61 @@
             "configuration": {
                 "type": "gpii.xrandr",
                 "capabilities": [
-                    "display.screenEnhacements",
+                    "display.screenEnhancement",
                     "applications.org\\.freedesktop\\.xrandr.id"
-                ]
-           }
+                ],
+                "capabilitiesTransformations": {
+                    "screen-resolution": {
+                        "transform": {
+                            "type": "fluid.transforms.valueMapper",
+                            "inputPath": "http://registry\\.gpii\\.net/common/screenResolution",
+                            "defaultInputValue": "normal",
+                            "options": {
+                                "high": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
+                                        }
+                                    }
+                                },
+                                "normal": {
+                                    "outputValue": {
+                                        "literalValue": {
+                                            "width": 1366,
+                                            "height": 768
```

Comment:
@javihernandez the windows version only sets values [that are available](https://github.com/OpenDirective/GPII-windows/blob/GPII-1000/gpii/node_modules/WindowsUtilities/WindowsUtilities.js#L689-L690). However, I know of no way to provide feedback to user if a selected setting is not available. I think we need to define the settings in ways that abstract sizes away so there can't be errors.